Aliases/ignition-tools2

```diff
@@ -1 +1 @@
-../Formula/ignition-tools.rb
```

Comment:
this alias should be deleted, sine there is now an ignition-tools2 formula

Reply:
https://github.com/osrf/homebrew-simulation/pull/1888/commits/cdfd567ea7eac97ff4dd046005076b185e05408b